alfredxml/type.go

```diff
@@ -90,6 +90,9 @@ type Data struct {
 	Clients     struct {
 		Num []ClientNum `xml:",any"`
 	} `xml:"clients"`
+	S2nProxy struct {
```

Comment:
This field should probably be called `S2NProxy`. See [Conventions](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).